test/test_jit.py

```diff
@@ -119,6 +119,9 @@ def canonical(graph):
 def LSTMCellF(input, hx, cx, *params):
     return LSTMCell(input, (hx, cx), *params)
 
+class Point(NamedTuple):
```

Comment:
I assume you define `Point` here because TorchScript can't compile class that's not defined in global scope? If so, you can actually define this class local to test case and use `make_global` to make TS see it.